internal/transport/transport_test.go

```diff
@@ -2656,7 +2656,7 @@ func TestConnectionError_Unwrap(t *testing.T) {
 	}
 }
 
-// Test that in the event of a graceful client transport shutdown, i.e.,
+// TestClientSendsAGoAwayFrame verifies that in the event of a graceful client transport shutdown, i.e.,
```

Comment:
The earlier comment was good enough. If at all you want to change it, s/Test/Tests/ would be more appropriate.

We have gone through this many times. Test comments can simply start with `Tests ...` instead of `TestFooBarAndBang ...` because the former conveys all the information that the latter does as well.